lefthook.yml

```
@@ -0,0 +1,6 @@
+pre-commit:
+  parallel: true
+  commands:
+    standardrb:
+      glob: "*.{rb}"
+      run: bundle exec standardrb
```

After #265 (Add Prettier) is merged
```suggestion
      run: bundle exec standardrb
    prettier:
      run: npx run prettier --write {staged_files} 
```